modules/core/src/main/java/org/apache/ignite/internal/util/IgniteUtils.java

```diff
@@ -51,6 +53,13 @@
 
     private static final boolean assertionsEnabled;
 
+    /**
+     * Gets the current monotonic time in milliseconds.
+     */
+    public static long monotonicMs() {
```

Comment:
Technically this is not true, nano time can in theory overflow and become negative. Why don't you use curentTimeMillis?

Reply:
Now the method is correct, thank you!